### PR TITLE
feat: add bookmarklet

### LINKED ISSUE #2: Implement bookmarklet

### PR DESCRIPTION
**Pull request by danschalow**
# Closes #2 

 - Open http://localhost:3000
 - Go to any article/term page
 - Paste content of `bookmarklet/bookmarklet.js` in URL bar

### CODE REVIEW

File: bookmarklet/bookmarklet.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+javascript: (function() {
+  const url =
+    "https://raw.githubusercontent.com/BurdaStyleGroup/edit-bookmarklet/master/bookmarklet/script/script.js";
+  fetch(url)
+    .then(response => response.text())
+    .then(response => eval(response));
```
Comment (tjwelde) [inline bookmarklet/bookmarklet.js]: using eval is discouraged https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval
Use `Function`, or just add the script via `<script>` tag to the DOM

Comment (tjwelde) [reply bookmarklet/bookmarklet.js]: I would prefer the script route

Comment (danschalow) [reply bookmarklet/bookmarklet.js]: I tried initially with the script route - the issue is, github don't return a valid `Content-Type` header.

I have implemented `Function`.

File: bookmarklet/bookmarklet.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+javascript: (function() {
+  const url =
+    "https://raw.githubusercontent.com/BurdaStyleGroup/edit-bookmarklet/57cf90f8b99de57a7a010586e6ec95ee11d19d44/bookmarklet/script/script.js";
```
Comment (danschalow) [inline bookmarklet/bookmarklet.js]: This still needs to be updated with `/master/bookmarklet/script/script.js` before we can merge.

File: bookmarklet/bookmarklet.js (hunk 1 above)
Comment (danschalow) [reply bookmarklet/bookmarklet.js]: Using `createElement` and setting `script.type = 'text/javascript'` unfortunately does not work.

**Comment by danschalow** [comment]
GitHub pages is the way to go. We can use this page too, for the editors to drag and drop their bookmarklet. 

In order to test properly, we need to get this PR merged, since GitHub pages watches the `master` branch of the project.

**Pull request merged**